src/pages/Home/CaregiverConnectionPage.tsx

```
@@ -17,25 +26,71 @@ const CaregiverConnectionPage = () => {
-      console.log('연결 코드:', connectionCode);
+      await createCaregiverLink({
+        phoneNumber: phoneWithoutHyphen,
+        code: parseInt(connectionCode, 10),
+      });
 
       // 성공 시 홈으로 이동
       navigate('/home');
-    } catch (error) {
-      console.error('연결 실패:', error);
-      // TODO: 에러 처리 (알림 등)
+    } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)